<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601916427342' stop='1601916441552'>
    <name>Add KNOS Relationship</name>
    <title>Add KNOS Relationship</title>
    <test-cases>
        <test-case start='1601916427342' status='passed' stop='1601916430204'>
            <name>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</name>
            <title>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='ad160d23-5c5d-48d3-8641-de6e7df71d08-attachment.png' type='image/png' size='51811'/>
            </attachments>
        </test-case>
        <test-case start='1601916430204' status='passed' stop='1601916434571'>
            <name>AddNewKNOS- It should add new casetype and subtype releationship to system!</name>
            <title>AddNewKNOS- It should add new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='480fca2d-420f-4ab6-ad1a-dd324174cd47-attachment.png' type='image/png' size='44290'/>
            </attachments>
        </test-case>
        <test-case start='1601916434571' status='failed' stop='1601916441552'>
            <name>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</name>
            <title>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d11ca0f5-270b-4ee8-800a-990481268c8d-attachment.png' type='image/png' size='57898'/>
            </attachments>
            <failure>
                <message>Expected 'Professional Tort' to equal 'CaseType123'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\UX013810\workspace\WCWRUI\Tests\Admin\UC_AD_16\AddNewKNOSTest.js:69:54)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>